src/adlmidi_private.hpp

```diff
@@ -277,6 +277,8 @@ class OPL3
     bool m_scaleModulators;
     //! Run emulator at PCM rate if that possible. Reduces sounding accuracy, but decreases CPU usage on lower rates.
     bool m_runAtPcmRate;
+    //! Enable soft panning
+    bool m_softPanning;
```

Comment:
You forgot to initialize it from the MIDIPlay's constructor where are all m_setup members are initialized...